app/data_models/session_data.py

```diff
@@ -1,50 +1,13 @@
-from typing import Any, Optional
+from typing import Optional
 
 
 class SessionData:
     def __init__(
         self,
-        tx_id: Optional[str] = None,
-        schema_name: Optional[str] = None,
-        period_str: Optional[str] = None,
         language_code: Optional[str] = None,
-        launch_language_code: Optional[str] = None,
-        ru_name: Optional[str] = None,
-        ru_ref: Optional[str] = None,
-        response_id: Optional[str] = None,
-        case_id: Optional[str] = None,
-        case_ref: Optional[str] = None,
-        account_service_base_url: Optional[str] = None,
-        account_service_log_out_url: Optional[str] = None,
-        trad_as: Optional[str] = None,
-        display_address: Optional[str] = None,
         confirmation_email_count: int = 0,
         feedback_count: int = 0,
-        schema_url: Optional[str] = None,
-        survey_url: Optional[str] = None,  # pylint: disable=unused-argument
-        **_: Any,
```

Comment:
This needs to stay. This is what makes it forwards compatible.

Reply:
Ok, tests removal reverted as well. Cheers!